src/main/java/com/rocketden/main/controller/v1/UserSocketController.java

```diff
@@ -0,0 +1,19 @@
+package com.rocketden.main.controller.v1;
+
+import com.rocketden.main.model.User;
+
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserSocketController extends BaseRestController {
+
+    @MessageMapping("/user")
+    @SendTo(BaseRestController.BASE_SOCKET_URL + "/subscribe-user")
```

Comment:
Question: I thought the config in `WebSocketConfig` meant you didn't need to specify the base path, but it seems like it's still necessary for the `@SendTo` annotations? I noticed the same thing in the `registerStompEndpoints` method as well; could you clarify when and where the effects of `configureMessageBroker` take place?

Reply:
That's a good question; the Spring tutorial gives the best example, I've copied and pasted the relevant material below:

Code:
```
@MessageMapping("/hello")
@SendTo("/topic/greetings")
public Greeting greeting(HelloMessage message) throws Exception {
    Thread.sleep(1000); // simulated delay
    return new Greeting("Hello, " + HtmlUtils.htmlEscape(message.getName()) + "!");
}
```
```
@Override
public void configureMessageBroker(MessageBrokerRegistry config) {
    config.enableSimpleBroker("/topic");
    config.setApplicationDestinationPrefixes("/app");
}
```

Explanation:
> The `configureMessageBroker()` method implements the default method in `WebSocketMessageBrokerConfigurer` to configure the message broker. It starts by calling `enableSimpleBroker()` to enable a simple memory-based message broker to carry the greeting messages back to the client on destinations prefixed with `/topic`. It also designates the `/app` prefix for messages that are bound for methods annotated with `@MessageMapping`. This prefix will be used to define all the message mappings. For example, `/app/hello` is the endpoint that the `GreetingController.greeting()` method is mapped to handle.

All of this is to say that I think, as far as I can understand, it's just syntax that the prefix for the `@MessageMapping(...)` value _doesn't_ include the prefix while the `@SendTo(...)` value _does_ include the prefix.